app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".controllers.MainActivity">
+
+    <TextView
+        android:id="@+id/textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="61dp"
+        android:layout_marginTop="16dp"
+        android:text="Welcome!"
+        app:layout_constraintBottom_toTopOf="@+id/login"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <Button
+        android:id="@+id/login"
+        android:layout_width="188dp"
+        android:layout_height="0dp"
```

Comment:
The height of the button should be `wrap_content` instead of `0dp` because having size of 0 cuts off the bottom part of the label of the button. This affects both the login and register buttons.
![screen shot 2018-09-24 at 2 15 42 pm](https://user-images.githubusercontent.com/36081199/45970585-ea69bb00-c004-11e8-8499-1244dd04395e.png)